core/src/main/java/io/grpc/ManagedChannelBuilder.java

```diff
@@ -180,6 +181,21 @@
   public abstract T compressorRegistry(CompressorRegistry registry);
 
   /**
+   * Set the duration for which the channel can stay without ongoing RPCs before going to idle
```

Comment:
"for which the channel can stay" can probably just be removed while also improving readability.